include/cling/Utils/Paths.h

```diff
@@ -0,0 +1,96 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author:  Jermaine Dupris <support@greyangle.com>
```

Comment:
No, just think everyone deserves more credit than they get...
If it really bothers you I'll pull it, but it makes me happy, and I'll be consistent.